app/store/utils/api.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+
+export default {
+  /**
+   * Runs a get request with given url with given url params.
+   * @param {String} url url to get to
+   * @param {Object} params url parameters which will be attached to the url
+   * @returns {Promise} promise with response data or error
+   */
+  get(url, params) {
+    return axios
+      .get(url, params)
+      .then(response => Promise.resolve(response.data))
```

Comment:
- [ ] I don't recommend to extract the data here. Maybe you will also need the message part. (e.g. form validation)